@here/harp-mapview-decoder/lib/TileLoader.ts

```diff
@@ -104,25 +104,19 @@ export class TileLoader {
     loadAndDecode(): Promise<TileLoaderState> {
         switch (this.state) {
             case TileLoaderState.Loading:
+            case TileLoaderState.Loaded:
```

Comment:
I looked into the code, it seems that the Loaded state is set for a very short time period. Maybe it would be worth at least adding a comment that this is potentially no longer necessary.